ext/native-decls/DoesTrainStopAtStations.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: CFX
+apiset: client
+game: gta5
+---
+## DOES_TRAIN_STOP_AT_STATIONS
+
+
+```c
+void DOES_TRAIN_STOP_AT_STATIONS(Vehicle train)
```

Comment:
Missing `;` natives won't compile. Same for `void SET_TRAIN_STOP_AT_STATIONS(Vehicle train, BOOL state)`

PS: such errors are usually visible when you run `.\prebuild.cmd`. E.g
```
25/809 - CREATE_RUNTIME_TEXTURE_FROM_DUI_HANDLE                                                     Error: C code snippet contained wrong amount of functions.
```

Reply:
I haven't been able to get prebuild.cmd to work on my machine. But corrected the issues in both native declaration and made sure prebuild.cmd was able to compile the natives on another machine